locales/zh_CN/LC_MESSAGES/gui.po

```diff
@@ -94,176 +96,184 @@ msgid ""
 "    gui.set_image(img)\n"
 "    gui.show(f'{frame:06d}.png')"
 msgstr ""
+"for frame in range(10000):\n"
+"    render(img)\n"
+"    gui.set_image(img)\n"
+"    gui.show(f'{frame:06d}.png')"
 
 #: ../../gui.rst:49
 msgid "Paint a window"
-msgstr ""
+msgstr "画一个窗户"
 
 #: ../../gui.rst:55
 msgid "(np.array or Tensor) tensor containing the image, see notes below"
-msgstr ""
+msgstr "（np.array或Tensor）张量包含图像，请参见下面的注释"
 
 #: ../../gui.rst:57
 msgid "Set a image to display on the window."
-msgstr ""
+msgstr "设置要在窗口上显示的图像。"
 
 #: ../../gui.rst:59
 msgid ""
-"The pixel, ``i`` from bottom to up, ``j`` from left to right, is set to "
-"the value of ``img[i, j]``."
-msgstr ""
+"The pixel, ``i`` from bottom to up, ``j`` from left to right, is set to the "
+"value of ``img[i, j]``."
+msgstr "从下到上的像素 ``i``，从左到右的像素 ``j`` 被设置为 ``img[i, j]`` 的值。"
 
 #: ../../gui.rst:62
 msgid "If the window size is ``(x, y)``, then the ``img`` must be one of:"
-msgstr ""
+msgstr "如果窗口大小是 ``(x, y)`` ，则 ``img`` 必须是以下之一："
 
 #: ../../gui.rst:64
 msgid "``ti.var(shape=(x, y))``, a grey-scale image"
-msgstr ""
+msgstr "``ti.var(shape=(x, y))`` ，灰度图像"
 
 #: ../../gui.rst:66
 msgid "``ti.var(shape=(x, y, 3))``, where `3` is for `(r, g, b)` channels"
-msgstr ""
+msgstr "``ti.var(shape=(x, y, 3))`` ，其中 `3` 代表 `(r, g, b)` 通道"
 
 #: ../../gui.rst:68
 msgid "``ti.Vector(3, shape=(x, y))`` (see :ref:`vector`)"
-msgstr ""
+msgstr "``ti.Vector(3, shape=(x, y))`` （参见 :ref:`vector`）"
 
 #: ../../gui.rst:70
 msgid "``np.ndarray(shape=(x, y))``"
-msgstr ""
+msgstr "``np.ndarray(shape=(x, y))``"
 
 #: ../../gui.rst:72
 msgid "``np.ndarray(shape=(x, y, 3))``"
-msgstr ""
+msgstr "``np.ndarray(shape=(x, y, 3))``"
 
 #: ../../gui.rst:75
 msgid "The data type of ``img`` must be one of:"
-msgstr ""
+msgstr " ``img`` 的数据类型必须是以下之一："
 
 #: ../../gui.rst:77
 msgid "float32, clamped into [0, 1]"
-msgstr ""
+msgstr "float32，固定为 [0, 1]"
 
 #: ../../gui.rst:79
 msgid "float64, clamped into [0, 1]"
-msgstr ""
+msgstr "float64，固定为 [0, 1]"
 
 #: ../../gui.rst:81
 msgid "uint8, range [0, 255]"
-msgstr ""
+msgstr "uint8，范围 [0, 255]"
 
 #: ../../gui.rst:83
 msgid "uint16, range [0, 65535]"
-msgstr ""
+msgstr "uint16，范围 [0, 65535]"
 
 #: ../../gui.rst:85
 msgid "uint32, range [0, UINT_MAX]"
-msgstr ""
+msgstr "uint32，范围 [0, UINT_MAX]"
 
 #: ../../gui.rst:91
 msgid "(tuple of 2) the position of circle"
-msgstr ""
+msgstr "（2元组）圆的位置"
 
 #: ../../gui.rst:92
 msgid "(optional, RGB hex) color to fill the circle"
-msgstr ""
+msgstr "（可选，RGB十六进制）颜色填充圆圈"
 
 #: ../../gui.rst:93 ../../gui.rst:103
 msgid "(optional, scalar) the radius of circle"
-msgstr ""
+msgstr "（可选，标量）圆的半径"
 
 #: ../../gui.rst:95
 msgid "Draw a solid circle."
-msgstr ""
+msgstr "画一个实心圆。"
 
 #: ../../gui.rst:101
 msgid "(np.array) the position of circles"
-msgstr ""
+msgstr "(np.array) 圆的位置"
```

Comment:
```suggestion
msgstr "（numpy 数组）一系列圆的位置"
```
Let's maintain the `s` in `circles`, so that user knows why it's a `numpy array` instead of a `tuple of 2`.